### PR TITLE
Aqa/js operators coercion

### PR DESCRIPTION
**Pull request by manatskaya**
This is pull request to lesson 4

### CODE REVIEW

File: js-operators-coercion/operatorsAndCoercion.js
Comment (landriiuk) [inline js-operators-coercion/operatorsAndCoercion.js:1]: Pls select one quote style and follow it for the course. single or double.
On a real project, there will be a code guide with recommendations on which one to use.

File: js-operators-coercion/operatorsAndCoercion.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+// task 4:
+let resource = "Wikipedia is a free online encyclopedia, created and edited by volunteers around the world and hosted by the Wikipedia Foundation."
+
+console.log('length of wikipedia sentence - ', resource.length);
+
+console.log('check end of the sentence - ', resource.endsWith('world'));
+
+console.log(resource.replaceAll('Wikipedia', 'Resource'));
+
```
Comment (landriiuk) [inline js-operators-coercion/operatorsAndCoercion.js:73]: there is missed part "- перевірте чи закінчується дана строка cловом "foundation""

File: js-operators-coercion/operatorsAndCoercion.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+console.log('rounding area of rectangle', areaOfRectangle.toFixed(2));
+
+let heightOfCylinder = 12;
+
+let volumeOfCylinder = p * (radiusOfCircle ** 2) * heightOfCylinder;
+
+console.log('calc volume of cylinder - ', volumeOfCylinder);
+
+console.log('rounding volume of cylinder', volumeOfCylinder.toFixed(2));
```
Comment (landriiuk) [inline js-operators-coercion/operatorsAndCoercion.js]: It's hard to read your code, small suggestions on how to improve it. It is called hoisting we will talk about it later.
```suggestion
let numberPI = Math.PI;

let radiusOfCircle = 5;

let lengthOfRectangle = Math.PI;

let widthOfRectangle = 7;

let heightOfCylinder = 12;

let areaOfCircle = numberPI * (radiusOfCircle ** 2);
console.log('calc area of circle - ', areaOfCircle);
console.log('rounding area of circle', areaOfCircle.toFixed(2));

let areaOfRectangle = lengthOfRectangle * widthOfRectangle;
console.log('calc area of rectangle - ', areaOfRectangle);
console.log('rounding area of rectangle', areaOfRectangle.toFixed(2));

let volumeOfCylinder = numberPI * (radiusOfCircle ** 2) * heightOfCylinder;
console.log('calc volume of cylinder - ', volumeOfCylinder);
console.log('rounding volume of cylinder', volumeOfCylinder.toFixed(2));
```

**Review by landriiuk — APPROVED** [review]

**Comment by landriiuk** [comment]
Added recommendations, your task is good.
pls add suggestion and merge this branch into main

File: js-operators-coercion/operatorsAndCoercion.js
Comment (manatskaya) [reply js-operators-coercion/operatorsAndCoercion.js:1]: okay, thank you

File: js-operators-coercion/operatorsAndCoercion.js (hunk 1 above)
Comment (manatskaya) [reply js-operators-coercion/operatorsAndCoercion.js:73]: added changes according to a comment

**Pull request merged**